platform-controller/src/main/java/org/hobbit/controller/tools/ProjectContentChecker.java

```diff
@@ -0,0 +1,97 @@
+package org.hobbit.controller.tools;
+
+import java.util.List;
+
+import org.apache.jena.query.Dataset;
+import org.apache.jena.query.DatasetFactory;
+import org.apache.jena.query.QueryExecution;
+import org.apache.jena.query.QueryExecutionFactory;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.StmtIterator;
+import org.hobbit.controller.gitlab.GitlabController;
+import org.hobbit.controller.gitlab.GitlabControllerImpl;
+import org.hobbit.controller.gitlab.Project;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ProjectContentChecker {
```

Comment:
This class is not covered with unit tests. Where is it used?

Reply:
Everyt class in the `tool` package is a simple cmd line tool that can be executed to check projects or search for a certain project or something similar. It is no functionality that is necessary for the platform but it is helpful for the administration or development.

I will add comments to all classes to emphasize their functionality. However, I wouldn't invest time into writing JUnit tests for them as long as the functionality is not needed for the rest of the platform.